site/en/guide/distributed_training.ipynb

```diff
@@ -130,7 +130,7 @@
         "| **Custom training loop** | Supported          | Supported     | Supported                     | Experimental support     | Experimental support      |\n",
         "| **Estimator API**        | Limited Support    | Not supported | Limited Support               | Limited Support          | Limited Support           |\n",
         "\n",
-        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibilities guarantees.\n",
+        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibily guarantees.\n",
```

Comment:
@synandi 

```suggestion
        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibility guarantees.\n",
```

@synandi Please be careful. Your fix contained a typo.

Reply:
@8bitmp3  acknowledged.